test/configurationTests.ts

```diff
@@ -26,7 +26,7 @@ import {
 import { IOptions } from "./../src/language/rule/rule";
 import { createTempFile } from "./utils";
 
-describe.only("Configuration", () => {
+describe("Configuration", () => {
```

Comment:
awkward. we should ban `describe.only` and `it.only` in this project's tslint config